src/net_processing.cpp

```diff
@@ -1775,8 +1775,10 @@ void PeerManagerImpl::RelayAddress(NodeId originator,
     // Relay to a limited number of other nodes
     // Use deterministic randomness to send to the same nodes for 24 hours
     // at a time so the m_addr_knowns of the chosen nodes prevent repeats
-    const uint64_t hashAddr{addr.GetHash()};
-    const CSipHasher hasher{m_connman.GetDeterministicRandomizer(RANDOMIZER_ID_ADDRESS_RELAY).Write(hashAddr).Write((GetTime() + hashAddr) / (24 * 60 * 60))};
+    const uint64_t hash_addr{CServiceHash(0, 0)(addr)};
```

Comment:
I think the SipHash vs SHA256 is a detail that need not be put in the commit message. It is really not relevant here. I changed it because we already have the `CServiceHash` which uses SipHash.

I played with `bench_bitcoin` and the `SHA256_32b` and `SipHash_32b` tests (the SipHash needs to be changed to `bench.batch(32).unit("byte").run(...` to print the same units - `ns/byte`). It looks like SipHash is about 7 times faster than the double-SHA256. But as you say, this code is not performance hot spot.